bower.json

```diff
@@ -5,7 +5,7 @@
     "url": "https://github.com/emberjs/ember.js.git"
   },
   "dependencies": {
-    "jquery": ">=1.11.1 <3.0.0",
```

Comment:
That's right. I rebased it as https://github.com/tricknotes/ember.js/commit/437726f63476e3ffd1ceabfeeb37b434b9fbdbb8 .